tests/parental_controls_gnome_software_oars.pm

```diff
@@ -85,17 +84,14 @@ sub run {
     # Try installing Cow’s Revenge from the command line. This should also fail.
     check_desktop_clean();
     type_very_safely("gnome-terminal\n");
-    assert_screen('desktop_terminal', 5);
+    sleep(5);
     type_string("gnome-software --install system/flatpak/flathub/desktop/com.github.dariasteam.cowsrevenge/stable\n");
```

Comment:
Would it make more sense to use `install_app` so we're testing that the Flatpak CLI enforces parental controls?

Reply:
No, this test is specifically to check that gnome-software doesn’t show an inappropriate app when run from the command line.